cypress/plugins/index.ts

```diff
@@ -23,13 +23,7 @@ export default async (on, config) => {
 
   if (isAPIMocked) {
     // TODO: Remove it after configuring mocks.
-    config.ignoreTestFiles = [
-      '**/integration/stories/opa.spec.ts',
-      '**/integration/stories/admin-settings/administrators.spec.ts',
-      '**/integration/stories/admin-settings/cluster-settings.spec.ts',
-      '**/integration/stories/admin-settings/custom-links.spec.ts',
-      '**/integration/stories/admin-settings/opa-integration.spec.ts',
-    ];
+    config.ignoreTestFiles = ['**/integration/stories/opa.spec.ts'];
```

Comment:
The machine deployment is disappearing after a short amount of time and therefor the test is failing, see [Image](https://minio.worker.ci.k8c.io/dashboard-cypress-artifacts/pr-3821-pre-dashboard-test-e2e-1456523126740357120/screenshots/stories/opa.spec.ts/OPA%20Story%20--%20should%20have%20healthy%20machine%20deployment%20%28failed%29.png) / [Video](https://minio.worker.ci.k8c.io/dashboard-cypress-artifacts/pr-3821-pre-dashboard-test-e2e-1456523126740357120/videos/stories/opa.spec.ts.mp4)
Not sure why this is happening, if you have any ideas feel free to adjust. In the meantime I just kept the story disabled.

You also then need to add 
```
"opaIntegration": {
  "enabled": true
},
```
to the clusters fixture, to enable opa properly.